CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+### 21.3.7 []()
+
+- Update openfisca_serve [rst](http://openfisca.readthedocs.io/en/latest/openfisca_serve.html) documentation
+  * Make obvious that gunicorn parameters can be used in `openfisca serve` command and explicit parameters' format
```

Comment:
I don't think `explicit` is a verb.  maybe `make xxx more explicit`or `explain` ?

Reply:
It is not, `make xxx more explicit` sounds good